### PR TITLE
Use tkn create pipeline in release.sh

### LINKED ISSUE #1174: Update release.sh to use tkn pipeline start
Currently, the [release script uses `kubectl`](https://github.com/tektoncd/cli/blob/ef620e0bbff28c68d6ae6decfe531d4763aa51b3/tekton/release.sh#L144) to create a PipelineRun for releasing `tkn`. We should have all the features in `tkn` to support using `tkn pipeline start`, and this will also help with dogfooding efforts for the project.

The command should be something like the following:
```
URL=$(git remote get-url ${PUSH_REMOTE}|sed 's,git@github.com:,https://github.com/,')
tkn pipeline start cli-release-pipeline -p revision=${RELEASE_VERSION} -p url=${URL} 
-w name=shared-workspace,volumeClaimTemplateFile=./volumeclaimtemplate.yaml
```

**volumeclaimtemplate.yaml:**

```yaml
spec:
  accessModes:
  - ReadWriteOnce
  resources:
    requests:
    storage: 500Mi
```

### PR DESCRIPTION
**Pull request by 06kellyjac**
# Changes

Resolves #1174

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Run the code checkers with `make check`
- [ ] Regenerate the manpages, docs and go formatting with `make generated`
- [X] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/cli/blob/master/CONTRIBUTING.md)
for more details._

# Release Notes

```release-note
NONE
```

### CODE REVIEW

**Comment by 06kellyjac** [comment]
/assign @chmouel

---

Not sure I can test this release.sh script myself

File: tekton/release.sh [hunk 1]
```diff
@@ -141,29 +141,11 @@ kubectl -n ${TARGET_NAMESPACE} apply -f ./tekton/release-pipeline.yml
...
-              storage: 500Mi
-  params:
-    - name: revision
-      value: ${RELEASE_VERSION}
-    - name: url
-      value: $(git remote get-url ${PUSH_REMOTE}|sed 's,git@github.com:,https://github.com/,')
-EOF
+URL=$(git remote get-url "${PUSH_REMOTE}" | sed 's,git@github.com:,https://github.com/,')
+# Start the pipeline
+tkn pipeline start cli-release-pipeline \
```
Comment (vinamra28) [inline tekton/release.sh:146]: Can we add -n ${TARGET_NAMESPACE} flag as well?

**Review by vinamra28 — COMMENTED** [review]
/ok-to-test

**Comment by chmouel** [comment]
oh cool thank you!

/lgtm

**Pull request merged**